### PR TITLE
Added package option details to dotnet list

### PR DESCRIPTION
**Pull request by anangaur**
## Summary

Added `package` option with `dotnet list`

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/core/tools/dotnet-list-reference.md [hunk 1]
```diff
@@ -39,8 +45,20 @@ The `dotnet list reference` command provides a convenient option to list project
   dotnet list app/app.csproj reference
   ```
 
+* List the package references for the specified project:
+
+  ```console
+  dotnet list package app/app.csproj reference
```
Comment (zivkan) [inline docs/core/tools/dotnet-list-reference.md]: package goes after the csproj, and remove the reference that's already after it

**Review by zivkan — COMMENTED** [review]
From memory, `dotnet list package` also supports the `.sln` file as an argument, then it will show package references from all projects in the sln.

Also, there are several list package specific arguments (transitive, outdated, others). Should these be documented?

File: docs/core/tools/dotnet-list-reference.md (hunk 1 above)
Comment (zivkan) [reply docs/core/tools/dotnet-list-reference.md]: in other words, change the line to
`dotnet list app/app.csproj package`

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks for your contribution @anangaur. But this should be added as a new topic along with https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package.

**Comment by anangaur** [comment]
I didn’t get a chance to revisit this. When do you plan to make your changes?

**Comment by mairaw** [comment]
Hopefully next week!

**Comment by anangaur** [comment]
Then yes. Please do add this in your changes too 🙂

**Comment by mairaw** [comment]
Closing since I've created #10227 with the new article. Please review.

**Pull request closed without merging**